test/passing/comments.ml.ref

```diff
@@ -269,3 +269,21 @@ class i =
     (* test *)
     inherit oo
   end
+
+let _ =
+  try_with (fun () ->
+      (* comment before *)
+      match get () with None -> do_something () | Some _ -> ())
+
+(* do nothing *)
+
+let _ = try_with (fun () -> (* comment before *)
```

Comment:
We're probably missing a break here.

Reply:
I'm having a look at this but this may take some time, maybe this PR could be approved first and we could fix this bug in another one?